### PR TITLE
[19.01] Use . activate "$GALAXY_CONDA_ENV"` to activate conda env

### PR DESCRIPTION
**Pull request by mvdbeek**
Fixes
https://github.com/galaxyproject/galaxy/issues/3876#issuecomment-463635193.
~~I'm not using `conda_activate` as I have a particular dislike for
putting random functions in my shell.~~

### CODE REVIEW

File: scripts/common_startup.sh [hunk 1]
```diff
@@ -123,7 +123,7 @@ if [ $SET_VENV -eq 1 -a $CREATE_VENV -eq 1 ]; then
                     $CONDA_EXE create --yes --override-channels --channel conda-forge --channel defaults --name "$GALAXY_CONDA_ENV" 'python=2.7' 'pip>=9' 'virtualenv>=16'
                     unset __CONDA_INFO
                 fi
-                conda_activate
+                . activate "$GALAXY_CONDA_ENV"
```
Comment (nsoranzo) [inline scripts/common_startup.sh]: That's what `conda_activate` does in `scripts/common_startup_functions.sh` if you are using bash. You may want to change `source` to `.` there, if that helps.

**Comment by gregvonkuster** [comment]
@mvdbeek Thanks so very much for all of the time you've spent on this!  And sorry to have kind of dumped it on you this morning.  :(

**Comment by mvdbeek** [comment]
No worries, it's good that we know this doesn't completely work as expected.

**Comment by mvdbeek** [comment]
So I had a `~/.profile` file that was getting activated somewhere along the way, and that put the wrong python binary on the PATH. Removing this everything works fine.

```
Found conda at: /home/mvandenb/miniconda3/bin/conda
Found Conda, will set up a virtualenv using conda.
To use a virtualenv instead, create one with a non-Conda Python 2.7 at .venv
Creating Conda environment for Galaxy: _galaxy_
To avoid this, use the --no-create-venv flag or set $GALAXY_CONDA_ENV to an
existing environment before starting Galaxy.
Collecting package metadata: done
Solving environment: done

## Package Plan ##

  environment location: /home/mvandenb/miniconda3/envs/_galaxy_

  added / updated specs:
    - pip[version='>=9']
    - python=2.7
    - virtualenv[version='>=16']


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    openssl-1.0.2p             |    h14c3975_1002         3.1 MB  conda-forge
    ------------------------------------------------------------
                                           Total:         3.1 MB

The following NEW packages will be INSTALLED:

  ca-certificates    conda-forge/linux-64::ca-certificates-2018.11.29-ha4d7672_0
  certifi            conda-forge/linux-64::certifi-2018.11.29-py27_1000
  libffi             conda-forge/linux-64::libffi-3.2.1-hf484d3e_1005
  libgcc-ng          conda-forge/linux-64::libgcc-ng-7.3.0-hdf63c60_0
  libstdcxx-ng       conda-forge/linux-64::libstdcxx-ng-7.3.0-hdf63c60_0
  ncurses            conda-forge/linux-64::ncurses-6.1-hf484d3e_1002
  openssl            conda-forge/linux-64::openssl-1.0.2p-h14c3975_1002
  pip                conda-forge/linux-64::pip-19.0.2-py27_0
  python             conda-forge/linux-64::python-2.7.15-h938d71a_1006
  readline           conda-forge/linux-64::readline-7.0-hf8c457e_1001
  setuptools         conda-forge/linux-64::setuptools-40.8.0-py27_0
  sqlite             conda-forge/linux-64::sqlite-3.26.0-h67949de_1000
  tk                 conda-forge/linux-64::tk-8.6.9-h84994c4_1000
  virtualenv         conda-forge/linux-64::virtualenv-16.0.0-py27_1000
  wheel              conda-forge/linux-64::wheel-0.33.0-py27_0
  zlib               conda-forge/linux-64::zlib-1.2.11-h14c3975_1004



Downloading and Extracting Packages
openssl-1.0.2p       | 3.1 MB    | ################################################################################################################### | 100%
Preparing transaction: done
Verifying transaction: done
Executing transaction: done
#
# To activate this environment, use:
# > source activate _galaxy_
#
# To deactivate an active environment, use:
# > source deactivate
#

Activating Conda environment: _galaxy_
WARNING: Your shell is not supported with Conda, attempting to use Conda env
         '_galaxy_' with manual environment setup. To avoid this
         message, use a supported shell or activate the environment before
         starting Galaxy.
New python executable in /tmp/galaxy/.venv/bin/python
copying /home/mvandenb/miniconda3/envs/_galaxy_/bin/python => /tmp/galaxy/.venv/bin/python
copying /home/mvandenb/miniconda3/envs/_galaxy_/bin/../lib/libpython2.7.so.1.0 => /tmp/galaxy/.venv/lib/libpython2.7.so.1.0
Installing setuptools, pip, wheel...done.
Activating virtualenv at .venv
DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 won't be maintained after that date. A future version of pip will drop support for Python 2.7.
Requirement already satisfied: pip>=8.1 in ./.venv/lib/python2.7/site-packages (19.0.2)
DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 won't be maintained after that date. A future version of pip will drop support for Python 2.7.
Looking in indexes: https://wheels.galaxyproject.org/simple, https://pypi.python.org/simple
```

**Pull request closed without merging**